sources/packages/api/src/route-controllers/designation-agreement/designation-agreement.controller.ts

```diff
@@ -17,21 +16,26 @@ import {
   UserToken,
 } from "../../auth/decorators";
 import { DesignationAgreementService, FormService } from "../../services";
-import { getISODateOnlyString, getUTCNow } from "../../utilities";
+import { getUTCNow } from "../../utilities";
 import {
   GetDesignationAgreementDto,
   GetDesignationAgreementsDto,
   SubmitDesignationAgreementDto,
 } from "./models/designation-agreement.model";
 import { InstitutionUserRoles } from "../../auth/user-types.enum";
 import { FormNames } from "../../services/form/constants";
-
+import { DesignationAgreementServiceController } from "./designation-agreement.service.controller";
+/***
+ * Designation agreement dedicated controller for Institution.
```

Comment:
doesn't the file name include `institution`?

Reply:
This controller was already existing before, so I did not rename it. We can talk about it.

Reply:
Agree, the client name should be included accordingly to what is already defined, in this case _institutions_.

Reply:
controller renamed.